sparksteps/steps.py

```diff
@@ -68,7 +78,7 @@ def cmd(self):
 
     @property
     def key(self):
-        return os.path.join(self.path, 'sources/', self.filename)
+        return os.path.join(self.path, self.filename)
```

Comment:
Perhaps we should explicitly use `posixpath.join` - I think the current implementation might be broken on windows due to different path separators.